static/app/components/replays/replayCountBadge.tsx

```diff
@@ -2,9 +2,6 @@ import Badge from 'sentry/components/badge';
 
 function ReplayCountBadge({count}: {count: undefined | number}) {
   const display = count && count > 50 ? '50+' : count ?? null;
-  if (display === null) {
-    return null;
-  }
```

Comment:
We shouldn't need this check anymore with the correct data fetching happening. 

The hooks will return `undefined` when data is still loading